libr/anal/p/anal_avr.c

```diff
@@ -1945,6 +1945,53 @@ static int archinfo(RAnal *anal, int q) {
 	return 2; // XXX
 }
 
+
+static ut8 *anal_mask_avr(RAnal *anal, int size, const ut8 *data, ut64 at) {
+	RAnalOp *op = NULL;
+	ut8 *ret = NULL;
+
+	if (!(op = r_anal_op_new ())) {
+		return NULL;
+	}
+
+	if (!(ret = malloc (size))) {
+		r_anal_op_free (op);
+		return NULL;
+	}
+
+	memset (ret, 0xff, size);
```

Comment:
why initialize to ff ? better use calloc and set the -1 where needed?

Reply:
it's initialized to `ff` because that's the "don't mask anything" value... which is fine for most instructions, only instructions with relocatable parts (like jumps or ptr loads) need a different one.

Reply:
ok